src/sinks/file/mod.rs

```diff
@@ -61,6 +61,7 @@ pub struct FileSinkConfig {
     #[serde_as(as = "serde_with::DurationSeconds<u64>")]
     #[serde(rename = "idle_timeout_secs")]
     #[configurable(metadata(docs::examples = 600))]
+    #[configurable(metadata(docs::human_name = "Idle Timeout"))]
     pub idle_timeout: Duration,
```

Comment:
I'm surprised this is needed, does this just become `Idle timeout` and we'd prefer the additional capitalization?

Reply:
@spencergilbert  Oh yeah, I meant to ask about this! I think [line 61](https://github.com/vectordotdev/vector/pull/17560/files#diff-6ecd088ef0da12f0cf2237692aa9a24cd920ab584a88019118b9b554b1636100L62) outputs the field name to be `Idle Timeout Secs` and I just wanted to remove `Secs`. Should I just remove line 61 instead?

Reply:
Ah, I missed the `rename` above. In that case this is correct, we want to keep the rename with `_secs` to keep in line with our [configuration spec](https://github.com/vectordotdev/vector/blob/master/docs/specs/configuration.md#option-naming) - thought _technically_ our spec says it should be `_seconds`.

Reply:
Should I update for the `rename` to be `idle_timeout_seconds`?

Reply:
I wouldn't worry about it here - it'd be a breaking change like that, and we aren't following that in so many places 😭, it might be less work to just update the spec.